src/styles/blocks/card__review-stars.scss

```diff
@@ -0,0 +1,18 @@
+.card {
+  &__review-stars {
+    display: flex;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
+      background-image: url('../images/star.svg');
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), nested selectors should be placed before any nested properties. Please move the `&:last-child` selector above the `background-image` property.